tests/translator/input/unsupported_resources.yaml

```diff
@@ -7,8 +7,8 @@ Resources:
       StageName: Prod
       DefinitionUri: s3://sam-demo-bucket/webpage_swagger.json
 
-  InvalidResourceTypeShouldBeAccepted:
-    Type: AWS::Serverless::ABCD
+  UnknownTypeShouldBeAccepted:
+    Type: AWS::Other::ABCD
```

Comment:
Should we have tests for both (`AWS::Other::ABCD` and `AWS::Serverless::ABCD`)? I am assuming the `AWS::Serverless::ABCD` was tested for a specific reason.

Reply:
Those are passing tests for the Translator.

We were accepting `AWS::Serverless::ABCD` before but we decided to validate the `AWS::Serverless::` resources with an allow list in the [validator](https://github.com/aws/serverless-application-model/pull/2032/files#diff-d0a88e876cff26b717361261093ba7d883eadeda19cb9504a4e8a1fc8b5ac185R78).

Reply:
I have mixed feelings about this. We aren't failing the Translator to start so fine place to start and see how things shake out. My worry is that someone is building a transform that uses `AWS::Serverless::` and while SAM doesn't support this, we risk failing in the future for something that would have worked.